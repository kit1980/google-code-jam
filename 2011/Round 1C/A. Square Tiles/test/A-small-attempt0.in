20
2 3
###
###
1 1
.
4 5
.##..
.####
.####
.##..
6 3
##.
##.
##.
##.
##.
##.
1 1
#
4 6
####..
######
######
####..
6 6
##.##.
##.##.
######
######
######
##..##
4 4
####
####
####
####
6 6
.####.
##..##
#.....
#..###
##...#
.####.
6 4
....
####
####
####
####
....
6 4
####
####
..##
####
####
..##
6 6
.#####
.#####
######
######
.##.##
.##.##
6 4
###.
###.
##..
####
####
##..
6 3
.##
.##
.##
.##
##.
##.
6 4
....
.##.
.##.
....
.##.
.##.
6 6
######
######
####..
####..
....##
....##
6 6
######
######
######
######
######
######
6 6
####..
####..
######
######
######
######
4 3
##.
##.
##.
##.
6 6
..##..
######
######
..##..
.####.
.####.
