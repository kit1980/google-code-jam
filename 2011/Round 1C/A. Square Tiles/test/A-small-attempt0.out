Case #1:
Impossible
Case #2:
.
Case #3:
./\..
.\//\
./\\/
.\/..
Case #4:
/\.
\/.
/\.
\/.
/\.
\/.
Case #5:
Impossible
Case #6:
/\/\..
\/\//\
/\/\\/
\/\/..
Case #7:
Impossible
Case #8:
/\/\
\/\/
/\/\
\/\/
Case #9:
Impossible
Case #10:
....
/\/\
\/\/
/\/\
\/\/
....
Case #11:
/\/\
\/\/
../\
/\\/
\//\
..\/
Case #12:
Impossible
Case #13:
Impossible
Case #14:
./\
.\/
./\
.\/
/\.
\/.
Case #15:
....
./\.
.\/.
....
./\.
.\/.
Case #16:
/\/\/\
\/\/\/
/\/\..
\/\/..
..../\
....\/
Case #17:
/\/\/\
\/\/\/
/\/\/\
\/\/\/
/\/\/\
\/\/\/
Case #18:
/\/\..
\/\/..
/\/\/\
\/\/\/
/\/\/\
\/\/\/
Case #19:
/\.
\/.
/\.
\/.
Case #20:
../\..
/\\//\
\//\\/
..\/..
./\/\.
.\/\/.
